docs/developer/tutorials/snmp/how-to.md

```diff
@@ -118,6 +118,115 @@ SNMP table: hrStorageTable
+
+The `snmpwalk` command can be used to perform a walk query.
+
+To facilitate usage of walk files for debugging, the following options are recommended: `-ObentU`. Here's what each option does:
+
+- `b`: do not break OID indexes down.
+- `e`: print enums numerically (for example, `24` instead of `softwareLoopback(24)`).
+- `n`: print OIDs numerically (for example, `.1.3.6.1.2.1.2.2.1.1.1` instead of `IF-MIB::ifIndex.1`).
+- `t`: print timeticks numerically (for example, `4226041` instead of `Timeticks: (4226041) 11:44:20.41`).
+- `U`: don't print units.
+
+For example, the following command gets a walk of the `1.3.6.1.2.1.1` (`system`) sub-tree:
+
+```console
+$ snmpwalk -v 2c -c public -ObentU 127.0.0.1:1161 1.3.6.1.2.1.1
+.1.3.6.1.2.1.1.1.0 = STRING: Linux 41ba948911b9 4.9.87-linuxkit-aufs #1 SMP Wed Mar 14 15:12:16 UTC 2018 x86_64
+.1.3.6.1.2.1.1.2.0 = OID: .1.3.6.1.4.1.8072.3.2.10
+.1.3.6.1.2.1.1.3.0 = 4226041
+.1.3.6.1.2.1.1.4.0 = STRING: root@localhost
+.1.3.6.1.2.1.1.5.0 = STRING: 41ba948911b9
+.1.3.6.1.2.1.1.6.0 = STRING: Unknown
+.1.3.6.1.2.1.1.8.0 = 9
+.1.3.6.1.2.1.1.9.1.2.1 = OID: .1.3.6.1.6.3.11.3.1.1
+.1.3.6.1.2.1.1.9.1.2.2 = OID: .1.3.6.1.6.3.15.2.1.1
+.1.3.6.1.2.1.1.9.1.2.3 = OID: .1.3.6.1.6.3.10.3.1.1
+.1.3.6.1.2.1.1.9.1.2.4 = OID: .1.3.6.1.6.3.1
+.1.3.6.1.2.1.1.9.1.2.5 = OID: .1.3.6.1.2.1.49
+.1.3.6.1.2.1.1.9.1.2.6 = OID: .1.3.6.1.2.1.4
+.1.3.6.1.2.1.1.9.1.2.7 = OID: .1.3.6.1.2.1.50
+.1.3.6.1.2.1.1.9.1.2.8 = OID: .1.3.6.1.6.3.16.2.2.1
+.1.3.6.1.2.1.1.9.1.2.9 = OID: .1.3.6.1.6.3.13.3.1.3
+.1.3.6.1.2.1.1.9.1.2.10 = OID: .1.3.6.1.2.1.92
+.1.3.6.1.2.1.1.9.1.3.1 = STRING: The MIB for Message Processing and Dispatching.
+.1.3.6.1.2.1.1.9.1.3.2 = STRING: The management information definitions for the SNMP User-based Security Model.
+.1.3.6.1.2.1.1.9.1.3.3 = STRING: The SNMP Management Architecture MIB.
+.1.3.6.1.2.1.1.9.1.3.4 = STRING: The MIB module for SNMPv2 entities
+.1.3.6.1.2.1.1.9.1.3.5 = STRING: The MIB module for managing TCP implementations
+.1.3.6.1.2.1.1.9.1.3.6 = STRING: The MIB module for managing IP and ICMP implementations
+.1.3.6.1.2.1.1.9.1.3.7 = STRING: The MIB module for managing UDP implementations
+.1.3.6.1.2.1.1.9.1.3.8 = STRING: View-based Access Control Model for SNMP.
+.1.3.6.1.2.1.1.9.1.3.9 = STRING: The MIB modules for managing SNMP Notification, plus filtering.
+.1.3.6.1.2.1.1.9.1.3.10 = STRING: The MIB module for logging SNMP Notifications.
+.1.3.6.1.2.1.1.9.1.4.1 = 9
+.1.3.6.1.2.1.1.9.1.4.2 = 9
+.1.3.6.1.2.1.1.9.1.4.3 = 9
+.1.3.6.1.2.1.1.9.1.4.4 = 9
+.1.3.6.1.2.1.1.9.1.4.5 = 9
+.1.3.6.1.2.1.1.9.1.4.6 = 9
+.1.3.6.1.2.1.1.9.1.4.7 = 9
+.1.3.6.1.2.1.1.9.1.4.8 = 9
+.1.3.6.1.2.1.1.9.1.4.9 = 9
+.1.3.6.1.2.1.1.9.1.4.10 = 9
+```
+
+As you can see, all OIDs that the device has available in the `.1.3.6.1.2.1.1.*` sub-tree are returned. In particular, one can recognize `sysObjectID` (line 2), `sysUpTime` (line 3), and `sysName` (line 5).
```

Comment:
(nit)

```suggestion
As you can see, all OIDs that the device has available in the `.1.3.6.1.2.1.1.*` sub-tree are returned. In particular, one can recognize:

- `sysObjectID` (`.1.3.6.1.2.1.1.2.0 = OID: .1.3.6.1.4.1.8072.3.2.10`)
- `sysUpTime` (`.1.3.6.1.2.1.1.3.0 = 4226041`)
- `sysName` (`.1.3.6.1.2.1.1.5.0 = STRING: 41ba948911b9`).
```

I think it's better to reference the element directly. Easier to read and more future proof than referencing lines.